functions/package.json

```diff
@@ -0,0 +1,43 @@
+{
+  "name": "functions",
+  "version": "0.0.1",
+  "scripts": {
+    "lint": "eslint \"src/**/*\"",
+    "lint:ci": "eslint \"src/**/*\" --max-warnings 0",
+    "test:ci": "echo \"Noting to do here\"",
+    "build": "yarn webpack",
+    "build:ci": "yarn build",
+    "serve": "npm run build && firebase emulators:start --only functions",
+    "shell": "npm run build && firebase functions:shell",
+    "start": "npm run shell",
+    "deploy": "firebase deploy --only functions",
+    "logs": "firebase functions:log"
+  },
+  "engines": {
+    "node": "12"
```

Comment:
It does not work to me, I have had to change it for 16.

Reply:
It's fine! you can use the even versions starting from the 10th!  [version 16 is in beta but supported by firebase](https://firebase.google.com/docs/functions/manage-functions#set_nodejs_version), so go ahead!

For anyone with the same problem, this version should be the same that is installed in your system, you can check the installed version with this command in the terminal:

`node -v`

I think that only even versions are supported because those are the LTS (long term support) ones